docs(portal-oidc) Notes on redirect_uri and developer accounts (#2100)
Kong/docs.konghq.com: app/enterprise/1.5.x/developer-portal/configuration/authentication/oidc.md

  "login_redirect_uri": ["http://localhost:8003"],
  "login_redirect_mode": "query"
}
```

The placeholders above should be replaced with your actual values:

  - `<CLIENT_ID>` - Client ID provided by IdP
  - `<CLIENT_SECRET>` - Client secret provided by IdP

See the [documentation of the OpenID Connect plugin](/hub/kong-inc/openid-connect/)
for more information.

**Important:** The `redirect_uri` needs to be configured as an allowed URI in the IdP.

> If not set explicitly in the configuration object it defaults to

Reviewer: ```suggestion
If not set explicitly in the configuration object, the URI default is
```
